doc/Makefile

```diff
@@ -154,4 +154,4 @@ doctest:
 	      "results in $(BUILDDIR)/doctest/output.txt."
 s3:
 	aws s3 sync _build/html s3://www.glueviz.org/en/`git symbolic-ref -q --short HEAD`
-	if [ `git symbolic-ref -q --short HEAD` == 'master' ]; then aws s3 sync _build/html s3://docs.glueviz.org/en/latest; fi
+	aws s3 sync _build/html s3://docs.glueviz.org/en/latest
```

Comment:
Also I think should be `www.glueviz.org` not `docs.glueviz.org` (this is a stale bug)